src/Support/Helpers/Report.php

```diff
@@ -10,14 +10,18 @@
      */
     function bugsnag_report(\Exception $exception)
     {
-        // Retrieve Bugsnag instance
-        $bugsnag = app('nodes.bugsnag');
-
-        // Report exception to Bugsnag
-        if ($exception instanceof \Nodes\Exceptions\Exception) {
-            $bugsnag->notifyException($exception, $exception->getMeta(), $exception->getSeverity());
-        } else {
-            $bugsnag->notifyException($exception, null, 'error');
+        try {
+            // Retrieve Bugsnag instance
+            $bugsnag = app('nodes.bugsnag');
+    
+            // Report exception to Bugsnag
+            if ($exception instanceof \Nodes\Exceptions\Exception) {
+                $bugsnag->notifyException($exception, $exception->getMeta(), $exception->getSeverity());
+            } else {
+                $bugsnag->notifyException($exception, null, 'error');
+            }   
+        } catch(\Exception $e) {
+            // Fail silent
```

Comment:
I think it's super important that the logging system does not crash the application.

And this happens all the time local so if we want a Log it should be if in production